hypothesis-python/docs/details.rst

```diff
@@ -18,8 +18,13 @@ Normally the output of a failing test will look something like:
 
 With the ``repr`` of each keyword argument being printed.
 
+<<<<<<< HEAD
+Sometimes this isn't enough, either because you have a value with a
+``__repr__()`` method that isn't very descriptive or because you need to see the output of some
+=======
 Sometimes this isn't enough, either because you have values with a ``repr`` that
 isn't very descriptive or because you need to see the output of some
+>>>>>>> Julian-O-stackoverflow
```

Comment:
You've committed the merge conflict markers here :wink:

Reply:
Huh, I've never made *that* particular ridiculous flub before. I guess that counts as learning!

Fixed. Thanks.

Reply:
No worries!  We should probably add a CI job which checks for the markers, but it's usually a SyntaxError in Python files so we haven't gotten around to it yet :slightly_smiling_face: